tests/epyccel/test_compare_expressions.py

```diff
@@ -6,9 +6,8 @@
 #==============================================================================
 class epyccel_test:
     """
-    Class to pyccelize module then compare different results
-    This avoids the need to pyccelize the file multiple times
-    or write the arguments multiple times
+    Class to pyccelize module or function then compare different results
+    This avoids the need to pyccelize the object multiple times
     """
     def __init__(self, f, lang='fortran'):
```

Comment:
I see that multiple files contain this class. Maybe we can move it to a separate "helper" file?

And by the way, this class cannot work for modules at the moment, only functions. If you want to use it for pyccelizing modules (and then compare functions therein) you will need to make it more complex.

Reply:
I think that would be helpful but I'm not sure where it makes sense to put that file. Should it be in the tests folder? Or in the pyccel folder?

> And by the way, this class cannot work for modules at the moment, only functions.

Ah you are right sorry. I don't think it should be used for modules as this may cause confusion about what is causing compiler failures and means that 1 failure in a module would make all tests reliant on that module fail

Reply:
> I'm not sure where it makes sense to put that file

I would put this class in the module `tests/epyccel/utilities.py`.



> I don't think it should be used for modules as this may cause confusion about what is causing compiler failures

I agree that it's better to only use this class for comparing functions. So please update its docstring accordingly.

Reply:
Done in 02e374d